samples/cpp/matchmethod_orb_akaze_brisk.cpp

```diff
@@ -0,0 +1,89 @@
+#include <opencv2/opencv.hpp>
+#include <vector>
+#include <iostream>
+
+using namespace std;
+using namespace cv;
+
+
+int main(void)
+{
+    vector<String> typeAlgoMatch;
+    typeAlgoMatch.push_back("BruteForce");
+    typeAlgoMatch.push_back("BruteForce-Hamming");
+    typeAlgoMatch.push_back("BruteForce-Hamming(2)");
+
+    vector<String> typeDesc;
+    typeDesc.push_back("AKAZE");
+    typeDesc.push_back("ORB");
+    typeDesc.push_back("BRISK");
+
+    String  dataFolder("../data/");
+    vector<String> fileName;
+    fileName.push_back("basketball1.png");
+    fileName.push_back("basketball2.png");
+
+    Mat img1 = imread(dataFolder+fileName[0], IMREAD_GRAYSCALE);
+    Mat img2 = imread(dataFolder+fileName[1], IMREAD_GRAYSCALE);
+
+    Ptr<Feature2D> b;
+
+    vector<String>::iterator itDesc;
+// Descriptor loop
+    for (itDesc = typeDesc.begin(); itDesc != typeDesc.end(); itDesc++){
+        Ptr<DescriptorMatcher> descriptorMatcher;
+        vector<DMatch> matches;	        /*<! Match between img and img2*/
+        vector<KeyPoint> keyImg1;		/*<! keypoint  for img1 */
+        vector<KeyPoint> keyImg2;		/*<! keypoint  for img2 */
+        Mat descImg1, descImg2;         /*<! Descriptor for img1 and img2 */
+        vector<String>::iterator itMatcher = typeAlgoMatch.end();
+        if (*itDesc == "AKAZE"){
+            b = AKAZE::create();
+        }
+        if (*itDesc == "ORB"){
+            b = ORB::create();
+        }
+        else if (*itDesc == "BRISK"){
+            b = BRISK::create();
+        }
+        try {
+            b->detect(img1, keyImg1, Mat());
+            b->compute(img1, keyImg1, descImg1);
+            b->detectAndCompute(img2, Mat(),keyImg2, descImg2,false);
+ // Match method loop
+             for (itMatcher = typeAlgoMatch.begin(); itMatcher != typeAlgoMatch.end(); itMatcher++){
+                descriptorMatcher = DescriptorMatcher::create(*itMatcher);
+                descriptorMatcher->match(descImg1, descImg2, matches, Mat());
+ // Keep best matches only to have a nice drawing
+                Mat index;
+                int nbMatch=int(matches.size());
+                Mat tab(nbMatch, 1, CV_32F);
+                for (int i = 0; i<nbMatch; i++)
+                    tab.at<float>(i, 0) = matches[i].distance;
```

Comment:
It seems useless, but I prefer to add brackets even with a single execution command. For new users it is quite confusing to not find these brackets.